.github/workflows/main.yaml

```diff
@@ -14,6 +14,13 @@ jobs:
       matrix:
         python-version: [3.7, 3.8, 3.9, "3.10", "3.11"]
         architecture: ["x64"]
+        include:
+          - python-version: 3.9
+            numpy-version: "2.0.dev"
+          - python-version: 3.10
+            numpy-version: "2.0.dev"
+          - python-version: 3.11
+            numpy-version: "2.0.dev"
```

Comment:
Per [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html), NumPy is only supported on Python 3.9+.